src/wp-admin/edit-form-advanced.php

```diff
@@ -543,8 +543,9 @@
 			$shortlink = wp_get_shortlink( $post->ID, 'post' );
 
 			if ( ! empty( $shortlink ) && $shortlink !== $permalink && home_url( '?page_id=' . $post->ID ) !== $permalink ) {
+				wp_enqueue_script( 'edit-events' );
```

Comment:
A new file just for this one button seems a bit too much.

Could this be added to `js/_enqueues/admin/post.js` instead?